route/build/src/pio_utils.f90

```diff
@@ -44,6 +46,11 @@ MODULE pio_utils
   public var_desc_t
   public io_desc_t
 
+  INTERFACE read_pnetcdf
```

Comment:
I talk about this above that pnetcdf in the name makes me think maybe you are accessing via pnetcdf directly. It might be better to name this closer to the underlying interfaces "read_array"...

Reply:
changed it to `read_dist_array`